### PR TITLE
add site speed testing tools

### PR DESCRIPTION
**Pull request by tcope25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,6 +307,11 @@ Table of Contents
   * [tesults.com](https://www.tesults.com) — Test results reporting and test case management. Integrates with popular test frameworks. Open Source software developers, individuals, educators, and small teams getting started can request discounted and free offerings beyond basic free project.
   * [cypress.io](https://www.cypress.io/) - Fast, easy and reliable testing for anything that runs in a browser. Cypress Test Runner is always free and open source with no restrictions and limitations. Cypress Dashboard is free for open source projects for up to 5 users.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  10 team members, Demo app and unlimited projects, 5,000 snapshots / month.
+  * [Pingdom](https://www.pingdom.com/) - Monitor your website's availability and performance for free with Pingdom and always be the first to know when your website is down.
```
Comment (ripienaar) [inline README.md]: they used to be free, but not any more

File: README.md [hunk 2]
```diff
@@ -307,6 +307,11 @@ Table of Contents
...
+  * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/) - PageSpeed Insights analyzes the content of a web page, then generates suggestions to make that page faster.
+  * [Speedmonitor.io](https://speedmonitor.io) - Track your website speed and performance data every day and track performance over time.
```
Comment (ripienaar) [inline README.md]: doesnt appear to be free?

File: README.md (hunk 1 above)
Comment (tcope25) [reply README.md]: I think you're right on this one.

File: README.md [hunk 3]
```diff
@@ -307,6 +307,11 @@ Table of Contents
   * [tesults.com](https://www.tesults.com) — Test results reporting and test case management. Integrates with popular test frameworks. Open Source software developers, individuals, educators, and small teams getting started can request discounted and free offerings beyond basic free project.
   * [cypress.io](https://www.cypress.io/) - Fast, easy and reliable testing for anything that runs in a browser. Cypress Test Runner is always free and open source with no restrictions and limitations. Cypress Dashboard is free for open source projects for up to 5 users.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  10 team members, Demo app and unlimited projects, 5,000 snapshots / month.
+  * [Pingdom](https://www.pingdom.com/) - Monitor your website's availability and performance for free with Pingdom and always be the first to know when your website is down.
+  * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/) - PageSpeed Insights analyzes the content of a web page, then generates suggestions to make that page faster.
```
Comment (tcope25) [inline README.md]: This is free provided by Google

File: README.md (hunk 2 above)
Comment (tcope25) [reply README.md]: It has a free tier like many of the other sites.

Comment (ripienaar) [reply README.md]: https://speedmonitor.io/pricing does not mention any free tier

**Comment by ripienaar** [comment]
thanks, feel free to reopen with only sites with free tiers

**Pull request closed without merging**